scipy/special/cephes/ndtr.c

```diff
@@ -147,6 +147,17 @@
 #include <float.h>		/* DBL_EPSILON */
 #include "mconf.h"
 
+#if !defined(__clang__) && defined(__GNUC__) && defined(__GNUC_MINOR__)
+#if __GNUC__ >= 5 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 4)
+#pragma GCC optimize("unroll-loops")
+#define cephes_isnan(x) __builtin_isnan(x)
```

Comment:
Does Numpy do this already?
https://github.com/numpy/numpy/blob/master/numpy/core/include/numpy/npy_math.h#L161